docs/change-log.md

```diff
@@ -2,6 +2,21 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update January 6th, 2023
+
+- **Improved:** More intuitive and easier to read design for table components across the application:
+    - Shared App Values & Secrets on [App Settings Screen](https://docs.humanitec.com/guides/orchestrate-infrastructure/manage-environment-variables-and-secrets#app-level);
+    - App Values & Secrets Overrides on [Envrionment Settings Screen](https://docs.humanitec.com/guides/orchestrate-infrastructure/manage-environment-variables-and-secrets#environment-level);
+    - [Variables](https://docs.humanitec.com/using-humanitec/work-with-workloads/work-with-environment-variables), [Files](https://docs.humanitec.com/using-humanitec/work-with-workloads/work-with-files), and [Volume Mounts](https://docs.humanitec.com/guides/orchestrate-infrastructure/manage-persistent-volumes) on [Workload Details Screen](https://docs.humanitec.com/reference/user-interface/workload-details-screen).
+
+- **Improved:** Responsive design of [Application Details Screen](https://docs.humanitec.com/reference/user-interface/app-details-screen) has been improved.
```

Comment:
I suggest to scratch "has been improved" at the end, as it reads like a duplication